integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -194,25 +194,35 @@ jobMaster:
 # UFS journal with HDFS example
 # journal:
 #   type: "UFS"
+#   ufsType: "non-local"
 #   folder: "hdfs://{$hostname}:{$hostport}/journal"
 # EMBEDDED journal to /journal example
 # journal:
 #   type: "EMBEDDED"
 #   folder: "/journal"
 journal:
-  type: "UFS" # "UFS" or "EMBEDDED"
-  ufsType: "local" # Ignored if type is "EMBEDDED". "local" or "HDFS"
-  folder: "/journal" # Master journal folder
+  # [ Required arguments ]
+  type: "UFS" # One of "UFS" or "EMBEDDED"
+  folder: "/journal" # Master journal directory or equivalent storage path
+  # [ Conditionally required arguments ]
+  #
+  # [[ UFS-backed journal options (journal.type="UFS") ]]
+  ufsType: "local" # One of "local" or "non-local"
```

Comment:
Can you add a comment here to explain the implications of `local`? This PR will be a good chance.

Something like `If "local", each master pod will have a PV for the journal`